README.md

```diff
@@ -25,5 +25,21 @@ Both of these features imply the use of [alloc](https://doc.rust-lang.org/alloc/
 
 To compile with `no_std`, disable default features via `--no-default-features` flag.
 
+## Testing
+
+You can use cargo defaults to test the library:
+
+```shell
+cargo test
+```
+
+However, some of the functions are heavy and might take a while for the tests to complete. In order to test in release mode, we have to replicate the same test conditions of the development mode so all debug assertions can be verified.
```

Comment:
nit: instead of "the same test conditions of the development mode" I would use either:
- "the same test conditions as in development mode"
- "the test conditions of the development mode"

It depends on the intended meaning

Reply:
Fixed!